en/views/helpers/form.rst

```diff
@@ -1544,28 +1544,24 @@ to your form's view template file::
     default values into input fields of type 'file'. Each time the form
     is displayed, the value inside will be empty.
 
-Upon submission, file fields provide an expanded data array to the
-script receiving the form data.
+To prevent the `submittedfile` from being over-written as blank, remove it from `$_accessible`.
+Alternatively, you can unset the index by using `beforeMarshal`
```

Comment:
```suggestion
Alternatively, you can unset the index by using ``beforeMarshal``::

```
You need to end the paragraph before a code block with `::` and a newline.

Reply:
Noted.